_targets/build.project.armv7a9-zynq7000

```diff
@@ -78,14 +78,24 @@ b_build_target() {
 		"${PREFIX_PROG_STRIPPED}plo-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" \
 		0
 
-	cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" _boot/
-	cp "${PREFIX_PROG_STRIPPED}phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf" _boot/
+	cp "${PREFIX_PROG_STRIPPED}plo-ram-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.img" "${PREFIX_BOOT}/"
+	cp "${PREFIX_PROG_STRIPPED}phoenix-${TARGET_FAMILY}-${TARGET_SUBFAMILY}.elf" "${PREFIX_BOOT}/"
+}
+
+
+b_flash_image() {
+	local flash_path="${PREFIX_BOOT}/flash-armv7a9-zynq7000.bin"
+
+	rm -f "${flash_path}"
+	dd if="${PREFIX_BOOT}/phoenix-armv7a9-zynq7000.disk" of="${flash_path}" bs=4M 2>/dev/null
+	truncate -s 16M "${flash_path}" 2>/dev/null
```

Comment:
please do note that "empty" flash should be filled with `0xFF` and not `0x00`. I assume this doesn't matter for now, but in the future the correct approach should be something like
```bash
dd if=/dev/zero bs=4M count=4 2>/dev/null | tr '\000' '\377' > "${flash_path}"
```

and then overwriting parts of it with desired contents with `conv=notrunc`

Reply:
If the future modification is needed, I will take yours proposition into account. Currently, I will leave it as it is.